src/main/java/no/unit/nva/model/Reference.java

```diff
@@ -2,13 +2,15 @@
 
 import com.fasterxml.jackson.annotation.JsonTypeInfo;
 import no.unit.nva.model.instancetypes.PublicationInstance;
+import nva.commons.utils.JacocoGenerated;
 
+import java.net.URI;
 import java.util.Objects;
 
 @JsonTypeInfo(use = JsonTypeInfo.Id.NAME, property = "type")
 public class Reference {
     private PublicationContext publicationContext;
-    private String doi;
+    private URI doi;
```

Comment:
We are now certain that a doi is a URI in all cases?

Reply:
Any reference in doi in datamodel is now a URI